src/app/shared/components/search/search.component.html

```diff
@@ -1 +1,4 @@
-<p>search works!</p>
+<form #searchForm="ngForm" class="search-form">
+  <input placeholder="{{placeHolder}}" class="search-input" #searchFraze>
+  <app-button (click)="sendSearchFraze(searchFraze.value)" [buttonText]="'Search'" class="search-button"></app-button>
+</form>
```

Comment:
The code looks good. I just want to show a bit alternative way using component class field:
```suggestion
<form class="search-form">
  <input placeholder="{{placeHolder}}" class="search-input" [(ngModel)]="searchStr">
  <app-button (click)="onSearchClick()" [buttonText]="'Search'" class="search-button"></app-button>
</form>
```
In component class:
```
searchStr: string = '';

onSearchClick() {
    this.newSearchEvent.emit(this.searchStr);
}
```